extensions/functions_logarithmic.yaml

```diff
@@ -5,6 +5,17 @@ scalar_functions:
     name: "ln"
     description: "Natural logarithm of the value"
     impls:
+      - args:
+          - name: x
+            value: i64
+        options:
+          rounding:
+            values: [ TIE_TO_EVEN, TIE_AWAY_FROM_ZERO, TRUNCATE, CEILING, FLOOR ]
+          on_domain_error:
+            values: [ NAN, "NULL", ERROR ]
```

Comment:
Domain error here is negative integers?

Reply:
yes, the domain error will be negative integers/floats